cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaBridgeAssemblyOperatorTest.java

```diff
@@ -605,6 +605,8 @@ public void testReconcileCallsCreateOrUpdate(VertxTestContext context) {
         // when requested ConfigMap for a specific Kafka Bridge cluster
         when(mockBridgeOps.get(eq(kbNamespace), eq("foo"))).thenReturn(foo);
         when(mockBridgeOps.get(eq(kbNamespace), eq("bar"))).thenReturn(bar);
```

Comment:
Both here and in the other classes -> do we still need to mock the `get()` methods? Or would `getAsync()` be enough?

Reply:
`getAsync` may be enough. I'll try it and update the PR if they're not necessary.

Reply:
Sync `get` methods have been removed.